src/connectors/ona-api/aggregateFormData.js

```diff
@@ -185,15 +214,21 @@ function processFormData(formData, indicatorField, aggregateOptions) {
           parsedLocName = parsedUID;
 
           if (extraProps && extraProps.length) {
-            extraProps.forEach((e, y) => {
-              extraPropsSumTotal[y] += parseInt(groupData[x][e] || 0, 10);
-            });
+            for (let y = 0; y < extraProps.length; y += 1) {
+              const e = extraProps[y];
+              if (numberProps.indexOf(e) !== -1) {
+                const int = parseInt(groupData[x][e] || 0, 10);
+                extraPropsSumTotal[e] += int;
```

Comment:
We can collapse this into one line and remove the `const inst`, that was just for debugging

Reply:
addressed